CSS/documentation.css

```diff
@@ -52,11 +53,11 @@ nav{
 /********************** content section of the page  ******************/
 
 main{
-    margin-top : 30px;
-    border : 3px solid red;
+    margin-top : 3rem;
+    /* border : 3px solid red; */
 }
 .content-section-container{
```

Comment:
there should be consistent spacing throughout the code. leave one line gap here.